pkg/compute/compute_integration_test.go

```diff
@@ -1678,6 +1775,34 @@ func makeDeployEnvironment(t *testing.T, manifestContent string) (rootdir string
 	return rootdir
 }
 
+func makePackEnvironment(t *testing.T, manifestContent string) (rootdir string) {
+	t.Helper()
+
+	rootdir, err := os.MkdirTemp("", "fastly-pack-*")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if err := os.MkdirAll(rootdir, 0700); err != nil {
+		t.Fatal(err)
+	}
```

Comment:
This is not needed - `os.MkdirTemp` will create the directory with 0700.

Reply:
Nice, thanks! Will remove.